src/main/java/me/xemor/skillslibrary2/effects/SpawnEffect.java

```diff
@@ -1,22 +1,24 @@
 package me.xemor.skillslibrary2.effects;
 
 import me.xemor.configurationdata.entity.EntityData;
-import me.xemor.skillslibrary2.SkillsLibrary;
 import org.bukkit.Location;
 import org.bukkit.configuration.ConfigurationSection;
 import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
 
 public class SpawnEffect extends Effect implements EntityEffect, TargetEffect, LocationEffect {
 
     private final EntityData entityData;
 
     public SpawnEffect(int effect, ConfigurationSection configurationSection) {
         super(effect, configurationSection);
-        ConfigurationSection entitySection = configurationSection.getConfigurationSection("entity");
-        if (entitySection == null) {
```

Comment:
We need this error message putting back in - users are noobs

Reply:
As it has a default entity even if a section isn't defined should I add "defaulted to zombie" to the message?

Reply:
oh true, it's up to you then. You don't **need** the error message if you've handled the null case